examples/drivenet/drivenet_close_loop.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DriveNet Close Loop Evaluation\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./drivenet_train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "## What is a close loop evaluation?\n",
+    "In close loop evaluation DriveNet is in **full control of the AV**. At each time step, we predict the future trajectory and then we move the AV int he first of the DriveNet's predictions. \n",
+    "\n",
+    "At Lyft, we refer to this process with the term **unroll**\n",
+    "\n",
+    "\n",
+    "## What is a good close loop metric?\n",
+    "For this setting metrics are particularly challenging. In fact, we would like to penalise some of the drifting (e.g. going off road or in the opposite lane) while at the same time allow others (e.g. different speed profiles)\n",
+    "\n",
+    "At Lyft L5, we use a substantial sets of different metrics to capture dangerous manoeuvres and behaviours. \n",
```

Comment:
Maybe could extend this a bit longer, introduce some more motivation?
E.g. our real measurement would be put it on car ... but costly ... so try to mimick real world and so on - this is why we do closed-loop.